code/api/src/modules/product/query.js

```diff
@@ -43,3 +43,12 @@ export const productTypes = {
   type: new GraphQLList(ProductTypesType),
   resolve: getTypes
 }
+
+// // Products for Survey
```

Comment:
What query will we use for the survey products in the place of this? 

edit: just saw the function down below! Never mind